library/src/main/scala/scala/scalajs/js/annotation/README.md

```diff
@@ -1,3 +0,0 @@
-**Attention**: Some files in here are also published in the Scala.js stubs JVM library (see the stubs project in the Scala.js build).
-
-If you add (or rename) a file, make sure the files in the stubs project are up to date.
```

Comment:
Shouldn't this file stay? I mean keeping the stubs in sync is still important.

Reply:
Done. I've also *added* an equivalent file in `reflect/annotation/README.md`.